dev-python/semantic_version/semantic_version-2.8.5.ebuild

```diff
@@ -13,7 +13,7 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 arm arm64 ~ppc ~ppc64 ~sparc x86"
+KEYWORDS="amd64 arm arm64 ~ppc ~ppc64 ~riscv ~sparc x86"
```

Comment:
Just to confirm, did you test it on some RISCV hardware or emulator?

Reply:
Initially added the keyword for the rust update if I remember correctly, but it doesn't matter now. It's a very simple python package that _should_ just work everywhere. That said, I'll refrain from touching keywords from now on and will remove this when submitting the fixed version.

Reply:
I've seen so many weird platform-specific Python failures I won't say 'should' ever again ;-).